runtime/src/invoke_dpe.rs

```diff
@@ -10,6 +10,14 @@ pub struct InvokeDpeCmd;
 impl InvokeDpeCmd {
     pub(crate) fn execute(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<MailboxResp> {
         if let Some(cmd) = InvokeDpeReq::read_from(cmd_args) {
+            // Validate data length
+            let Ok(in_len) = usize::try_from(cmd.data_size) else {
```

Comment:
It looks like we actually don't use `cmd.data_size`. I think we need to update the execute call to `.execute_serialized_command(&mut env, drivers.mbox.user(), &cmd.data[..cmd.data_size as usize])`

Reply:
Done.